assets/js/index.js

```diff
@@ -0,0 +1,123 @@
+---
+---
+
+// generic App class
+class App {
+  constructor(container) {
+    this._init(container);
+    this._bindEvents();
+  }
+
+  _init(container) {
+    this.appContainer = container;
+    // this.currentSections = this.appContainer.querySelectorAll('[data-section]');
+    this.mobileNavIcon = this.appContainer.querySelectorAll('[data-mobile-nav-icon]');
+    this.mobileNav = this.appContainer.querySelector('[data-mobile-menu]');
+  }
+
+  _bindEvents() {
+    this.mobileNavIcon.forEach(icon => {
+      icon.addEventListener('click', () => {
+        this.toggleMobileMenu();
+      })
+    })
+  }
+
+  toggleMobileMenu() {
+    $(this.mobileNav).toggleClass('MobileMenu--opened');
+  }
+}
+
+// SubNav class
+class SubNav {
+  constructor() {
+    this._init();
+    this._bindEvents();
+  }
+
+  _init() {
+    this.subNavItems = app.appContainer.querySelectorAll('[data-sub-nav-item]');
+  }
+
+  _bindEvents() {
+    this.subNavItems.forEach(item => {
+
+      item.addEventListener("click", (e) => {
+        e.preventDefault();
+        this.scrollToSection(item.getAttribute('data-sub-nav-item'))
```

Comment:
Let's add a static `selectors` method to his class for stuff like `data-sub-nav-item` since it's getting used multiple times.